app/javascript/helpers/ajaxRequestHelper.js

```diff
@@ -0,0 +1,73 @@
+import {XMLHttpRequest} from 'xmlhttprequest';
```

Comment:
Not sure why we are using this package for request, it's quite outdated. I think it would be better to use browsers [fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) or modern library like [axios](https://github.com/axios/axios)

Reply:
Well noted if I need that one day! Thanks!